### PR TITLE
Change name from CiliumEndpointBatch to CiliumEndpointSlice

### PR DESCRIPTION
**Pull request by krishgobinath**
1) Change name of CEB object to CES i.e. CiliumEndpointBatch to CiliumEndpointSlice.

Signed-off-by: Gobinath Krishnamoorthy <gobinathk@google.com>

### CODE REVIEW

**Comment by Weil0ng** [comment]
seems like you forgot to include updated agent cmd-ref?

File: operator/pkg/ciliumendpointbatch/endpointbatch.go [hunk 1]
```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package ciliumendpointbatch
+package ciliumendpointslice
```
Comment (Weil0ng) [inline operator/pkg/ciliumendpointbatch/endpointbatch.go:15]: name of the file and its path should be changed too

Comment (krishgobinath) [reply operator/pkg/ciliumendpointbatch/endpointbatch.go:15]: > name of the file and its path should be changed too

Intentionally left out for now, by renaming all the files in a folder, would erase all history of a file. Once it passes all criteria to merge then we will change.

**Review by aanm — APPROVED** [review]

**Review by ldelossa — APPROVED** [review]
like this change :+1:

**Review by Weil0ng — APPROVED** [review]

**Pull request merged**